### PR TITLE
Fix example code for React

### PR DESCRIPTION
**Pull request by mraible**
Fixes the following stack trace when running the latest Create React App:

```
console.error node_modules/react-dom/cjs/react-dom.development.js:506
  Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
      in Home (created by Secure)
      in Secure (created by withAuth(Home))
      in withAuth(Home) (created by Route)
      in Route (at App.js:18)
      in div (created by Security)
      in Security (created by Route)
      in Route (created by withRouter(Security))
      in withRouter(Security) (at App.js:17)
      in Router (created by BrowserRouter)
      in BrowserRouter (at App.js:16)
      in App (at App.test.js:7)
```

### CODE REVIEW

**Comment by nbarbettini** [comment]
@robertjd This seems like a super simple fix, do we need to run through some UAT on our side?

File: _source/quickstart-fragments/react/default-example.md [hunk 1]
```diff
@@ -102,8 +101,12 @@ export default withAuth(class Home extends Component {
     }
   }
 
-  componentDidUpdate() {
-    this.checkAuthentication();
+  async componentDidMount() {
+    await this.checkAuthentication();
```
Comment (bretterer) [inline _source/quickstart-fragments/react/default-example.md]: Why are we changing this to await?  What is this accomplishing?

File: _source/quickstart-fragments/react/default-example.md [hunk 2]
```diff
@@ -147,10 +150,7 @@ class App extends Component {
   render() {
     return (
       <Router>
-        <Security issuer={config.issuer}
-                  client_id={config.client_id}
-                  redirect_uri={config.redirect_uri}
-        >
+        <Security {...config}>
```
Comment (bretterer) [inline _source/quickstart-fragments/react/default-example.md]: For the example, I am not sure doing this should be done.  It does not explain what config options are available or expected.

**Comment by bretterer** [comment]
Thank you for this PR to help improve our documentation. As you may know, we have moved our developer documentation to a new repo.

Since this PR is targeting our developer docs, we are requesting that you migrate this PR over to the new repo https://github.com/okta/okta-developer-docs

If you have any questions about this, or need assistance with a complex PR, please let us know in #eng-devex

**Pull request closed without merging**